hbase-server/src/main/java/org/apache/hadoop/hbase/master/cleaner/CleanerChore.java

```diff
@@ -213,9 +213,14 @@ private void preRunCleaner() {
 
   public Boolean runCleaner() {
```

Comment:
Can be boolean here? We will not return null in the future.get?

Reply:
Yeah , it can be.